variables.js

```diff
@@ -0,0 +1,17 @@
+function variables() {
+
+    let userinput;
+    userinput = document.getElementById("user").value;
+
+    let passinput;
+    passinput = document.getElementById("pass").value;
+
+    let GFG_Var;
+ 
+    userdisplay = (userinput);
+    document.getElementById('result').innerHTML = `${userdisplay}` ;
+    }
+
+function redirect() {
+    window.location.href = "file:///C:/Users/Samardh/Desktop/Health%20App/code.html" ;
```

Comment:
This href will break when on anyone else's computer, as the filepath is specific to your computer. You should do research on how redirect links in projects can be formatted such that they'll work on any computer.

Reply:
used the actual file instead code.html